pkg/storage/fs.go

```diff
@@ -32,6 +33,24 @@ import (
 
 const privateKeyFilePostfix = "private.pem"
 
+type FileOpenError struct {
+	filePath 	string
+	legalEntity string
```

Comment:
Inconsistent indentation... gofmt snafu?

Reply:
This triggered me to update some of my editor settings to do auto gofmt and goimports